lib/core/versions/latest/DocumentComposer.ts

```diff
@@ -249,15 +248,28 @@ export default class DocumentComposer {
     }
   }
 
-  private static validateServiceEndpointId (serviceEndpointId: any) {
-    if (typeof serviceEndpointId !== 'string') {
-      throw new SidetreeError(ErrorCode.DocumentComposerPatchServiceEndpointIdNotString);
+  private static validateId (id: any) {
+    if (typeof id !== 'string') {
+      throw new SidetreeError(ErrorCode.DocumentComposerIdNotString);
```

Comment:
Nit: include the input in the error message? The tests will have to change to use the JasmieSidetreeErrorValidator class though so up to you if the change is big.

Reply:
Updated.